mlx_lm/models/openelm.py

```diff
@@ -44,6 +44,8 @@ def make_divisible(
     Returns:
         new_v: new divisible value
     """
+    if divisor is None:
+        raise ValueError("divisor must not be None")
```

Comment:
I would just change the type here from `Optional[int]` to `int` and remove this line.